{{ cookiecutter.repo_name }}/{{ cookiecutter.pkg_name }}/config/settings/dev.py

```diff
@@ -39,5 +40,5 @@ def DEVSERVER_DEFAULT_ADDR(self):
     )
 
     # Beware before activating this! Grappelli has problems with admin
-    # inlines and TEMPLATE_STRING_IF_INVALID.
-    TEMPLATE_STRING_IF_INVALID = values.Value('')
+    # inlines and the template backend option 'string_if_invalid'.
+    Common.TEMPLATES[0]['OPTIONS']['string_if_invalid'] = 'INVALID EXPRESSION %s'
```

Comment:
The `string_if_invalid` option should be moved to the `Common.TEMPLATES` setting. It should be controllable using an environment variable named `TEMPLATE_STRING_IF_INVALID`. It's default should be an empty string. Please don't forget to move the comment regarding Grappelli too.